zppy/templates/coupled_global.py

```diff
@@ -0,0 +1,872 @@
+# Script to plot some global atmosphere and ocean time series
```

Comment:
This looks like it got added back in for some reason in the rebase. Delete this; it's in zppy-interfaces now.